File: in_classs/in_classs/ViewController.m
//
//  ViewController.m
//  in_classs
//
//  Created by Sherwin on 2020-03-16.
//  Copyright © 2020 Sherwin. All rights reserved.
//

#import "ViewController.h"
#import <AVFoundation/AVFoundation.h>
@interface ViewController ()
{
    AVAudioPlayer *_audioClip;
    
}
@end


@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    NSString *path = [NSString stringWithFormat:@"%@/song1.mp3",[[NSBundle mainBundle] resourcePath]];
                  NSURL *soundUrl = [NSURL fileURLWithPath:path];
                      
                      _audioClip = [[AVAudioPlayer alloc] initWithContentsOfURL:soundUrl error:nil];
}

- (IBAction)playound:(id)sender {
    
    [_audioClip play];
    
}



- (IBAction)sliderValue:(id)sender {
    


}
@end

File: inclass22/inclass22/ViewController.swift
//
//  ViewController.swift
//  inclass22
//
//  Created by Sherwin on 2020-01-30.
//  Copyright © 2020 Sherwin. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
  
    
    @IBAction func showAlert() {
        
        let alertController = UIAlertController(title: "Disclaimer", message: "Before using this teaching", preferredStyle: .alert)
        
        let declineAction = UIAlertAction(title: "Decline", style: .cancel, handler: nil)
        let acceptAction = UIAlertAction(title: "Accept", style: .default) { (_) -> Void in
            
            self.performSegue(withIdentifier: "somesegue", sender: self) // Replace SomeSegue with your segue identifier (name)
        }
        
        alertController.addAction(declineAction)
        alertController.addAction(acceptAction)
        
        present(alertController, animated: true, completion: nil)
    }

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }


}

